packages/common/src/webrtc/VertoHandler.ts

```diff
@@ -31,7 +31,7 @@ class VertoHandler {
 
     if (dialogId && session.dialogs.hasOwnProperty(dialogId)) {
       if (attach) {
-        session.dialogs[dialogId].hangup()
+        // session.dialogs[dialogId].hangup()
```

Comment:
Do we need this hangup using the lib with a standalone FS/mod_verto? 

Could be something like that?

```suggestion
      if (!this.nodeId) {
        session.dialogs[dialogId].hangup()
      }
```